### PR TITLE
Add more info to the package upload section

### PR DESCRIPTION
**Pull request by minhqdao**
- [x] Add more info to the package upload section.
- [x] Add fpm version requirement.
- [x] Change `--show-form-data` to `--show-upload-data`.

### CODE REVIEW

**Review by awvwgk — APPROVED** [review]

File: pages/spec/publish.md [hunk 1]
```diff
@@ -30,10 +34,10 @@ A package must specify a valid [SPDX](https://spdx.org/licenses/) license in its
 
 ### Upload rights
 
-A package can only be uploaded by package admins and maintainers. Package admins can grant admin and maintainer rights to other users.
+A package can only be uploaded by package admins and maintainers. Package admins can grant admin and maintainer rights to other users to upload and maintain individual packages. Namespace admins can grant admin and maintainer rights to other users to upload to that namespace and maintain all packages of that namespace.
```
Comment (arteevraina) [inline pages/spec/publish.md:37]: A package can only be uploaded by namespace admins and namespace maintainers. 
Namespace admins or maintainers can grant package maintainer rights to other users to upload and maintain individual packages. Namespace admins can grant admin and maintainer rights to other users to upload to that namespace and maintain all packages of that namespace.

**Review by arteevraina — CHANGES_REQUESTED** [review]
Looks good to me except one minor change.

**Comment by perazz** [comment]
Agree on @arteevraina's suggestion. lgtm otherwise.

**Review by perazz — APPROVED** [review]

File: pages/spec/publish.md (hunk 1 above)
Comment (minhqdao) [reply pages/spec/publish.md:37]: Ok, so package maintainers cannot upload packages? What can they do then?

Comment (arteevraina) [reply pages/spec/publish.md:37]: Package maintainers can maintain individual packages which means they can upload the new versions of the individual packages. 

* Adds users as maintainers to the package.
* Generate upload token for a package.
* Upload new version of the existing package.

**Review by arteevraina — APPROVED** [review]

**Pull request merged**